tests/test_trusted_metadata_set.py

```diff
@@ -129,6 +129,21 @@ def test_update(self):
 
         self.assertTrue(count, 6)
 
+    def test_update_metadata_output(self):
+        timestamp = self.trusted_set.update_timestamp(self.metadata["timestamp"])
+        snapshot = self.trusted_set.update_snapshot(self.metadata["snapshot"])
+        self.trusted_set.update_targets(self.metadata["targets"])
+        delegeted_targets_1 = self.trusted_set.update_delegated_targets(
+            self.metadata["role1"], "role1", "targets"
+        )
+        delegeted_targets_2 = self.trusted_set.update_delegated_targets(
+            self.metadata["role2"], "role2", "role1"
+        )
+        self.assertIsInstance(timestamp, Metadata)
+        self.assertIsInstance(snapshot, Metadata)
+        self.assertIsInstance(delegeted_targets_1, Metadata)
+        self.assertIsInstance(delegeted_targets_2, Metadata)
```

Comment:
```suggestion
        self.assertIsInstance(timestamp, Timestam)
        self.assertIsInstance(snapshot, Snapshot)
        self.assertIsInstance(delegeted_targets_1, Targets)
        elf.assertIsInstance(delegeted_targets_2, Targets)
```
I think it makes sense to be more precise about the types in these asserts.

Reply:
I don't think this suggestion is correct. You'd have to check `timestamp.signed` type

Reply:
You are correct, then I will fix my suggestion to:
```suggestion
        self.assertIsInstance(timestamp.signed, Timestam)
        self.assertIsInstance(snapshot.signed, Snapshot)
        self.assertIsInstance(delegeted_targets_1,.signed Targets)
        elf.assertIsInstance(delegeted_targets_2.signed, Targets)
```
which would be better to check the `signed` type instead of the generic `Metadata` type.